izanami-frontend/src/pages/users.tsx

```diff
@@ -217,8 +229,69 @@ export function Users() {
             </div>
           </>
         )}
+        <div
+          className={`d-flex align-items-center ${
+            hasSelectedRows ? "" : "invisible"
```

Comment:
Right now select stays visible when all selected rows have disappeared from screen after delete, which is inconsistent since we want Select to be visible only when there is selected row